rosbag2_tests/test/rosbag2_tests/test_rosbag2_cpp_get_service_info.cpp

```diff
@@ -309,8 +312,6 @@ TEST_P(Rosbag2CPPGetServiceInfoTest, get_service_info_for_bag_with_topics_and_se
   EXPECT_TRUE(ret) << "Failed to capture " << expected_messages << " expected messages in time";
 
   recorder->stop();
-  stop_spinning();
-  cleanup_process_handle.cancel();
```

Comment:
Suggestion to return deleted explict call to `stop_spinning()` and `cleanup_process_handle.cancel();`

Reply:
Same comment as https://github.com/ros2/rosbag2/pull/1796#discussion_r1752020122